api/src/extensions/manager.ts

```diff
@@ -307,10 +308,10 @@ export class ExtensionManager {
 
 		logger.info('Watching extensions for changes...');
 
-		const extensionDirUrl = pathToRelativeUrl(getExtensionsPath());
+		const extensionsDir = path.posix.resolve(getExtensionsPath());
```

Comment:
Do we need `path.posix.resolve` here? This function is known to have some footguns resulting in non-posix compliant paths on Windows in certain situations (for example `path.posix.resolve(process.cwd(), 'foo') -> '/Users/Brainslug/C:\\Users\\Brainslug/foo'`)

Reply:
The usage of posix (213 and 236) could be causing the issue in https://github.com/directus/directus/issues/18721. We should check if this is resolved once paths are made OS agnostic.

Reply:
Nope, that's https://github.com/paulmillr/chokidar/issues/1051

Reply:
It might not be as that one reports it as a macOS issue while #18721 reports that it works on macOS/linux but not windows.

Reply:
Hmm, I cannot imagine how that could affect the Docker container, as the OS will always be Linux.

But as outlined in https://github.com/directus/directus/pull/21111#discussion_r1455898895, happy to switch to OS agnostic `path.resolve`.
@nickrum Do you remember the reason for using `path.posix`?

Reply:
Yup, I just answered [here](https://github.com/directus/directus/pull/21111#discussion_r1456064371) before I saw your comment 😅